cdk_python/src/detectors/python-cdk-auto-scaling-group-scaling-notifications/python-cdk-auto-scaling-group-scaling-notifications_compliant.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-cdk-auto-scaling-group-scaling-notifications@v1.0 defects=0}
+import aws_cdk as cdk
+from aws_cdk.aws_autoscaling import AutoScalingGroup, ScalingEvents,ScalingEvent, NotificationConfiguration
+from aws_cdk.aws_sns import Topic, TopicEncryption
+from aws_cdk.aws_ec2 import InstanceClass,InstanceType, MachineImage, Vpc
+
+class CdkStarter(cdk.Stack):
+    def __init__(self, scope: cdk.App, id: str):
+        super(scope, id)
+
+        negTopic=Topic(self, 'rAsgTopic', encryption=TopicEncryption.KMS)
+        # Compliant: Covers notifications for events other than error events.
```

Comment:
what the notification event should be or best to have for the ASG, could you pls have a code example and annotation to list all the required or suggested notification event.

Reply:
Sure, I'll add the example. Thanks!